migrations/20180727141620_initial_setup.js

```diff
@@ -12,6 +12,7 @@ exports.up = function(knex, Promise) {
     .dropTableIfExists('authorisations')
     .dropTableIfExists('establishments')
     .raw('create extension if not exists "uuid-ossp"')
+
```

Comment:
I _know_ this is just stylistic, but we should probably leave the migration files as read only

Reply:
Yeah this was just helping me read it while I was using the migrations to create the schemas.

Can revert if required?

Reply:
Yeah, I agree with @joefitter - let's be super strict about changes to migration files.